wiper.bash

```diff
@@ -92,29 +92,48 @@ T_ids_wrong_files_validation_containers-json() {
 
 T_ids_wrong_files_validation_curl-weave() {
   local result=0
-  ids_file_validation test/curl-weave || result=${?}
+  weave_ids_file_validation test/curl-weave || result=${?}
   [[ ${result} != 0 ]]
 }
 
 T_ids_wrong_files_validation_weave-with-expose() {
   local result=0
-  ids_file_validation test/weave-with-expose || result=${?}
+  weave_ids_file_validation test/weave-with-expose || result=${?}
   [[ ${result} != 0 ]]
 }
 
 T_ids_wrong_files_validation_emptyfile() {
   local result=0
-  ids_file_validation test/emptyfile || result=${?}
+  docker_ids_file_validation test/emptyfile || result=${?}
   [[ ${result} != 0 ]]
 }
 
 T_ids_file_validation() {
   local result=0
-  ids_file_validation test/docker-ids || result=${?}
+  docker_ids_file_validation test/docker-ids || result=${?}
   [[ ${result} == 0 ]]
 }
 
-ids_file_validation() {
+weave_ids_file_validation() {
```

Comment:
Could we keep the original function, but just change the bit that checks for an empty file:

if docker id is zero length -> error and return 1
if weave id is zero length -> warn and return 0

Reply:
Done!